common/model-views.cpp

```diff
@@ -4143,10 +4143,14 @@ namespace rs2
         ImGui::OpenPopup(title.c_str());
         if (ImGui::BeginPopupModal(title.c_str(), NULL, ImGuiWindowFlags_AlwaysAutoResize))
         {
+            if (error_message != "")
+                ImGui::CloseCurrentPopup();
```

Comment:
Explain

Reply:
ImGui get confused when a popup is opened within a popup.
This way if an error popup comes up the current popup is closed.
Sergey advised me to use this method.

Reply:
So explain it in the code in such a way that a reader would understand.

Reply:
Removed it for now, didn't fix the pop up issue anyway.